lib/cli/src/commands/run.rs

```diff
@@ -586,6 +557,19 @@ impl Run {
             .collect::<Result<Vec<_>>>()?;
         Ok(func.call(ctx, &invoke_args)?)
     }
+}
+
+impl Run {
+    /// Executes the `wasmer run` command
+    pub fn execute(&self) -> Result<(), anyhow::Error> {
+        // downloads and installs the package if necessary
```

Comment:
I think we said that we always need to prefer local files if they exist, and only then check for remote packages.

Just needs an additional check here if the file exists, and then prefer the local file.

Reply:
Ok, done in https://github.com/wasmerio/wasmer/pull/3416/commits/575373c55a0a1f98212a6dd20f3b37cfccf255a4